### PR TITLE
Pricing update

### PR DESCRIPTION
**Pull request by yashwin**

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -2148,3 +2148,44 @@ $(document).ready(function() {
 		closeEffect	: 'none'
 	});
 	});
+
+
+
+$(document).ready(function(){
```
Comment (subho007) [inline js/main.js]: **[eslint]**
- Error - 'document' is not defined. (no-undef)

File: js/main.js [hunk 2]
```diff
@@ -2148,3 +2148,44 @@ $(document).ready(function() {
...
+
+    $("#monthly").click(function(){
+				var title = $( ".calculatePrice" ).attr( "value" );
+			$(".calculatePrice").text("US $" + title + "/month");
+
+			var title = $( ".calculatePrice2" ).attr( "value" );
```
Comment (subho007) [inline js/main.js]: **[eslint]**
- Error - 'title' is already defined (no-redeclare)

File: js/main.js [hunk 3]
```diff
@@ -2148,3 +2148,44 @@ $(document).ready(function() {
...
+		$(".calculatePrice2").text("US $" + title + "/month");
+		var title = $( ".calculatePrice3" ).attr( "value" );
```
Comment (subho007) [inline js/main.js]: **[eslint]**
- Error - 'title' is already defined (no-redeclare)

File: js/main.js [hunk 4]
```diff
@@ -2148,3 +2148,44 @@ $(document).ready(function() {
...
+	$(".calculatePrice3").text("US $" + title + "/month");
+    });
+    $("#yearly").click(function(){
+				var value = $( ".calculatePrice" ).attr( "value" );
+				var total =value*10;
+        $(".calculatePrice").text("US $" + total + "/year");
+				var value = $( ".calculatePrice2" ).attr( "value" );
```
Comment (subho007) [inline js/main.js]: **[eslint]**
- Error - 'value' is already defined (no-redeclare)

File: js/main.js [hunk 5]
```diff
@@ -2148,3 +2148,44 @@ $(document).ready(function() {
...
+				var total =value*10;
```
Comment (subho007) [inline js/main.js]: **[eslint]**
- Error - 'total' is already defined (no-redeclare)

File: js/main.js (hunk 2 above)
Comment (subho007) [inline js/main.js]: **[jshint]**
-  'title' is already defined.

File: js/main.js [hunk 6]
```diff
@@ -2148,3 +2148,44 @@ $(document).ready(function() {
...
+				$(".calculatePrice2").text("US $" + total + "/year");
+				var value = $( ".calculatePrice3" ).attr( "value" );
```
Comment (subho007) [inline js/main.js]: **[eslint]**
- Error - 'value' is already defined (no-redeclare)

File: js/main.js (hunk 3 above)
Comment (subho007) [inline js/main.js]: **[jshint]**
-  'title' is already defined.

File: js/main.js [hunk 7]
```diff
@@ -2148,3 +2148,44 @@ $(document).ready(function() {
...
+				var total =value*10;
```
Comment (subho007) [inline js/main.js]: **[eslint]**
- Error - 'total' is already defined (no-redeclare)

File: js/main.js (hunk 4 above)
Comment (subho007) [inline js/main.js]: **[jshint]**
-  'value' is already defined.

File: js/main.js (hunk 5 above)
Comment (subho007) [inline js/main.js]: **[jshint]**
-  'total' is already defined.

File: js/main.js [hunk 8]
```diff
@@ -2148,3 +2148,44 @@ $(document).ready(function() {
...
+				$(".calculatePrice3").text("US $" + total + "/year");
+    });
+
+});
+
+
+$(document).ready(function(){
```
Comment (subho007) [inline js/main.js:2152]: **[eslint]**
- Error - 'document' is not defined. (no-undef)

File: js/main.js (hunk 6 above)
Comment (subho007) [inline js/main.js]: **[jshint]**
-  'value' is already defined.

File: js/main.js (hunk 7 above)
Comment (subho007) [inline js/main.js]: **[jshint]**
-  'total' is already defined.

**Comment by subho007** [comment]
@yashwin there are some more changes, please talk to Manik to get those 👍

File: js/main.js (hunk 1 above)
Comment (subho007) [inline js/main.js]: @yashwin ignore this

File: js/main.js (hunk 2 above)
Comment (subho007) [inline js/main.js]: @yashwin there are problem in this code, a lot of variable redefining is happening

File: js/main.js (hunk 3 above)
Comment (subho007) [inline js/main.js]: @yashwin there are problem in this code, a lot of variable redefining is happening

File: js/main.js (hunk 4 above)
Comment (subho007) [inline js/main.js]: @yashwin there are problem in this code, a lot of variable redefining is happening

File: js/main.js (hunk 6 above)
Comment (subho007) [inline js/main.js]: @yashwin there are problem in this code, a lot of variable redefining is happening

**Comment by yashwin** [comment]
Yeah. I will talk to him.

File: js/main.js (hunk 8 above)
Comment (subho007) [inline js/main.js:2152]: @yashwin you can ignore this

**Comment by subho007** [comment]
@yashwin also look at the circle-ci and see what error it is giving

File: js/main.js (hunk 6 above)
Comment (yashwin) [inline js/main.js]: I am working on this. I will use Jekyll here.

File: js/main.js [hunk 9]
```diff
@@ -2148,3 +2148,18 @@ $(document).ready(function() {
 		closeEffect	: 'none'
 	});
 	});
+
+$(document).ready(function(){
```
Comment (subho007) [inline js/main.js:2152]: **[eslint]**
- Error - 'document' is not defined. (no-undef)

**Comment by subho007** [comment]
@yashwin you have to also look at the CircleCI errors. did you see the build is failing? check this: https://circleci.com/gh/appknox/appknox.github.io/48

**Comment by yashwin** [comment]
Checking that @subho007

File: js/main.js [hunk 10]
```diff
@@ -2148,3 +2148,20 @@ $(document).ready(function() {
...
+				$(".yearly-plan").hide();
+				$(".monthly-plan").show();
+				$(".hide-savings").removeClass("show-savings");
+    });
+		$(".year").click(function(){
+				$(".year").addClass("blue");
+				$(".month").removeClass("blue");
+				$(".monthly-plan").hide();
+				$(".yearly-plan").show();
+			  $(".hide-savings").addClass("show-savings");
```
Comment (subho007) [inline js/main.js]: **[eslint]**
- Error - Mixed spaces and tabs. (no-mixed-spaces-and-tabs)

Comment (subho007) [inline js/main.js]: @yashwin fix this error. Only use space not tabs

**Pull request merged**